src/Engine/GameEvents/LambdaEvent.java

```diff
@@ -0,0 +1,49 @@
+package Engine.GameEvents;
+
+import java.util.function.Consumer;
+
+import Engine.XYCoord;
+import Terrain.MapMaster;
+import UI.MapView;
+import UI.Art.Animation.GameAnimation;
+
+public class LambdaEvent implements GameEvent
```

Comment:
Obviously you know I have to object to this on principle.
I can understand there are reasons you would want to have a catch-all event type, but those are the precise reasons I would use to argue against having a catch-all event type.

Reply:
Check out how it's used in Bear&Bull.
Without making a whole new event type for *each CO*, there's no way to accomplish setting up event sequences like that.
You could argue, maybe, that I should have implemented differently, but that was the simplest option I could see.